core/domain/suggestion_services.py

```diff
@@ -3936,22 +3937,30 @@ def _generate_translation_contributor_certificate_data(
     words_count = 0
     for model in suggestions:
         suggestion = get_suggestion_from_model(model)
+        suggestion_change = suggestion.change
+        if (
+                suggestion_change.cmd == 'add_written_translation' and (
+                    translation_domain.TranslatableContentFormat
+                    .is_data_format_list(suggestion_change.data_format)
+                )
+        ):
+            words_count += len(suggestion_change.translation_html)
```

Comment:
Presumably the translation here is a list of strings? If so then will len() give the correct value? Better if you do something like sum(len(item) for item in translation_html), IMO

Reply:
Done